repose-aggregator/components/filters/api-validator-filter/src/main/java/org/openrepose/filters/apivalidator/ApiValidatorHandler.java

```diff
@@ -153,9 +146,16 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
                 }
 
                 if (!isValid) {
-                    if (mbcsInvalidRequests != null) {
+                    if (Optional.ofNullable(metricsService).isPresent()) {
+                        MetricRegistry metricRegistry = metricsService.getRegistry();
                         for (String s : matchedRoles) {
-                            mbcsInvalidRequests.mark(s);
+                            metricRegistry.meter(
```

Comment:
Remember that this call potentially invokes a bunch of synchronization in the Dropwizard metrics library. Since we want this meter for every request, we may as grab it on initialization and keep track of it. Following that pattern, we also only ever need to call `metricsService.getRegistry()` once, during initialization, when we grab all of the metrics we are going to need.